### PR TITLE
Fix EnsembleForecaster and AutoEnsembleForecaster docstring example

### PR DESCRIPTION
**Pull request by aiwalter**

### CODE REVIEW

File: sktime/forecasting/compose/_ensemble.py [hunk 1]
```diff
@@ -61,16 +61,17 @@ class AutoEnsembleForecaster(_HeterogenousEnsembleForecaster):
...
     >>> from sktime.forecasting.naive import NaiveForecaster
     >>> from sktime.forecasting.trend import PolynomialTrendForecaster
     >>> from sktime.datasets import load_airline
     >>> y = load_airline()
-    >>> forecasters = [("trend", PolynomialTrendForecaster()),\
-                        ("naive", NaiveForecaster())]
-    >>> forecaster = AutoEnsembleForecaster(forecasters=forecasters, n_jobs=2)
+    >>> forecasters = [
+    ...     ("trend", PolynomialTrendForecaster()),
+    ...     ("naive", NaiveForecaster())]
```
Comment (fkiraly) [inline sktime/forecasting/compose/_ensemble.py:73]: if you want this filly linted, the last `]` should be on a new line and line 73 should end in `,`

**Review by fkiraly — DISMISSED** [review]
looks good to me, except for one minor linting issue

**Review by mloning — APPROVED** [review]

**Review by GuzalBulatova — APPROVED** [review]
Your changes make sense, thank you! Looks all good to me.

**Comment by aiwalter** [comment]
@fkiraly pls approve and merge

**Comment by mloning** [comment]
Now merged - thanks everyone! :)

**Pull request merged**